### ISSUE TITLE
Unable to run upgrade on one cluster, status code 413.

### ISSUE THREAD

**Issue by chris-bosman** [helm/helm#4471]
I have multiple k8s clusters that I run in Azure. I was previously deploying this one with kubectl apply commands, but am trying to bring it in line with my other k8s clusters and deploy it using helm upgrade. I was able to perform helm install on this cluster, but helm upgrade fails every time with the following error: 

`Error: UPGRADE FAILED: the server responded with the status code 413 but did not return more information (post configmaps)`

All I can find regarding this error is related to nginx ingress issues. The nginx-ingress *is* installed in this cluster, but it is not part of the chart deployment that I'm upgrading. Additionally, there are no configmaps that are part of this chart. Looking at the logs of the tiller pod reveal no more information than the above error message.

As this is hampering our deployment strategy of this application, any timely assistance on this matter would be much appreciated.

Thank you.

Output of `helm version`:
Client: &version.Version{SemVer:"v2.10.0-rc.3", GitCommit:"9ad53aac42165a5fadc6c87be0dea6b115f93090", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.10.0-rc.3", GitCommit:"9ad53aac42165a5fadc6c87be0dea6b115f93090", GitTreeState:"clean"}

(NOTE: This error was also present in v2.10.0-rc.2)

Output of `kubectl version`: 
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T22:29:25Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.5", GitCommit:"32ac1c9073b132b8ba18aa830f46b77dcceb0723", GitTreeState:"clean", BuildDate:"2018-06-21T11:34:22Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
AKS

**Comment by chris-bosman** [comment]
This was user error. The chart I was deploying had grown too large and the Tiller-created configmap that holds the chart info couldn't be posted. It had grown too large because I hadn't cleaned out the tar files from previous versions.

**Comment by prajnadnayak** [comment]
I am facing the same issue. I tried to use both ConfigMaps and Secrets but still get the 413 error. 

Error: the server responded with the status code 413 but did not return more information (post configmaps)
helm.go:75: [debug] the server responded with the status code 413 but did not return more information (post configmaps)

The size of the chart is less than 1MB.  I have deleted and recreated the namespace where the release was deployed just to make sure the incremented release history is not taking up the space. I am not sure what else to check. Any help is appreciated.
